### PR TITLE
show winners

### PR DESCRIPTION
**Pull request by jeremybanka**
- **💄 staging**
- **➕ framer motion**
- **✨ display first keyframes**

### CODE REVIEW

File: public/image-caching-worker.js
Comment (psbanka) [inline public/image-caching-worker.js:1]: I've never done this before! Good idea

File: src/components/SeeResults/SeeResults.tsx [hunk 1]
```diff
@@ -384,32 +391,260 @@ function SeeResults(): JSX.Element {
...
+			</nav>
+			<aside>
+				{electionRef.current ? <ElectionRounds election={electionRef.current} /> : null}
+			</aside>
 		</div>
 	)
 }
 
+export const Phases = {
+	surplus_allocation({ children }) {
```
Comment (psbanka) [inline src/components/SeeResults/SeeResults.tsx:443]: why `snake_case` here?

File: src/components/SeeResults/SeeResults.tsx [hunk 2]
```diff
@@ -384,32 +391,260 @@ function SeeResults(): JSX.Element {
...
+			<motion.div layoutId="keyframe" data-keyframe="distribute_surplus">
+				<header>distribute surplus</header>
+			</motion.div>
+		)
+	},
+	// winner_selection
+	show_candidates({ election }) {
+		const view = useO(resultsViewAtom)
+		const currentRound = election.rounds[view.round]
+		// biome-ignore lint/style/noNonNullAssertion: we swag this
```
Comment (psbanka) [inline src/components/SeeResults/SeeResults.tsx:515]: `swag` ?

File: src/components/SeeResults/SeeResults.tsx [hunk 3]
```diff
@@ -384,32 +391,260 @@ function SeeResults(): JSX.Element {
...
+									total={total}
+								/>
+							)
+						})}
+					</ol>
+				</main>
+			</motion.div>
+		)
+	},
+
```
Comment (psbanka) [inline src/components/SeeResults/SeeResults.tsx:536]: remove newline?

File: src/index.scss [hunk 4]
```diff
@@ -7,28 +7,25 @@
 }
 
 body {
-  background-image: url("./bg.jpg");
```
Comment (psbanka) [inline src/index.scss:10]: should probably remove `public/bg.jpg` as part of this PR too?

**Review by psbanka — APPROVED** [review]
Wow, this has come a long way! Beautiful! 🎉

File: src/components/SeeResults/SeeResults.tsx (hunk 2 above)
Comment (jeremybanka) [reply src/components/SeeResults/SeeResults.tsx]: haha sorry just my default comment when I'm working through a temporary thing. should all be changed now

File: src/components/SeeResults/SeeResults.tsx (hunk 1 above)
Comment (jeremybanka) [reply src/components/SeeResults/SeeResults.tsx:446]: good q. This is generally how I format magic strings. However, these became a lot more than that...

File: public/image-caching-worker.js
Comment (jeremybanka) [reply public/image-caching-worker.js:1]: me neither! But the AI knew exactly what to do! Solves the very real issue of the images flickering between frames/renders

**Pull request merged**